### DISCUSSION TITLE
How to add options in the drop down list for 'front/rear port' type?

### DISCUSSION THREAD

**Discussion by ghost** [netbox-community/netbox#9095] [Q&A] [answered]
Is there a way to natively (without code modification, plugins, or a feature request) add options to the drop down list for rear port 'types' ?
Currently there are quite a few options already (8P8C, 110 Punch, LC, MPO, Splice, etc.) but is there a way to add additional types?
Same applies to the types for front ports as well as power port.

**Comment by candlerb** [comment, marked as answer]
No. Netbox maintains a fixed list of these, and if you want to propose a new connector type then you have to raise it as a feature request on Github.

What connector type are you thinking of?

For interfaces there is type "Other". However this doesn't appear to be offered for Front Port or Rear Port.  I thought this had been requested as a feature some time ago, but the nearest I can find is #6564.

For power ports there is no type "Other", but you can leave the type unset.

**Reply by ghost** [reply]
Issue raised : https://github.com/netbox-community/netbox/issues/9098
